Swift/Document.swift

```diff
@@ -33,6 +33,10 @@ public class Document : DictionaryProtocol, Equatable, Hashable, Sequence {
         return impl.revisionID
     }
     
+    public var timestamp: Double {
```

Comment:
Should it be NSTimeInterval instead of Double?

Reply:
TimeInterval specifically for Swift.

Reply:
Indeed, will change to TimeInterval